listener/upload.go

```diff
@@ -10,71 +10,83 @@ import (
 	"crypto/sha256"
 	"encoding/hex"
 	"encoding/json"
-	"fmt"
 	"github.com/RedHatInsights/patchman-clients/inventory"
 	"github.com/RedHatInsights/patchman-clients/vmaas"
 	"github.com/pkg/errors"
+	"github.com/segmentio/kafka-go"
 	"time"
 )
 
-func uploadHandler(event mqueue.PlatformEvent) {
-	tStart := time.Now()
-	defer utils.ObserveSecondsSince(tStart, messageHandlingDuration.WithLabelValues(EventUpload))
+type Host struct {
+	ID                    string                    `json:"id,omitempty"`
```

Comment:
As we need only ID, SystemProfile and culling related fields, we can ignore the others, can't we?

Reply:
Yes, but I'd rather follow the spec here, providing we might need some of these later

Reply:
I don't think so. It's better to implement only what you need. It's no problem add it later once you really need it (probably never :smile: ).